docs/proposals/kubeedgeadm-scope.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Kubeedgeadm kubeEdge installer scope (Issue 324)
+status: Pending
+authors:
+    - "@samy2019"
+approvers:
+  - "@m1093782566"
+  - "@rohitsardesai83"
+  - "@sids-b"
+creation-date: 2019-04-11
+last-updated: 2019-04-12
+---
+
+## Motivation
+Many users shared their feedback that kubeEdge installation is too complicated and it may prevent people from trying kubeEdge.
+There should a simplified way to have **Getting Started with KubeEdge**, so that user can concentrate more on using it instantly, rather than
+getting entangled in the installation steps.
+
+## Proposal
+KubeEdge shall have simple commands and steps to bring up both cloud and edge components.
+The user experience in **Getting Started with KubeEdge** will be seamless.
+Hence proposing the following commands for KubeEdge installation process.
+
+For cloud, commands shall be:
+
+- `kubeedge cloud init`
+- `kubeedge cloud reset`
+
+For edge, commands shall be:
+
+- `kubeedge node init`
```

Comment:
I don't think we need `init` for node at all. Please note that the only motivation is that we want to simply the installation for users. I don't think it's worth building a complicated tool.